### ISSUE TITLE
Shortened Training Time on Larger Dataset: Suspected Issue in Dataloader Iteration

### ISSUE THREAD

**Issue by OswaldoBornemann** [k2-fsa/icefall#1420]
When I want to train on a new dataset (which is larger than the previous dataset) based on a previous checkpoint, it used to take 4-5 days to train for one epoch. However, I've found that now it takes only about 2 days to train for one epoch. Therefore, I suspect there might be a problem somewhere in the dataloader's iteration in the training, causing the model to not actually traverse all the data during the actual training

**Comment by OswaldoBornemann** [comment]
I just tried to comment the following in the `load_checkpoint_if_available` function and see whether this would solve the problems.

**Comment by OswaldoBornemann** [comment]
I have tried to refer to the `finetune.py`, and tried to replace the `load_checkpoint_if_available` with `load_model_params`. I think most previous changes from my side is almost the same as `load_model_params`. Except for omitting the loading of `optimizer` and `scheduler`.

**Comment by OswaldoBornemann** [comment]
The same thing happens. So before I used the `lhoste==1.0.5` (This May), now i replace with `lhoste==1.18`. Is the latest lhoste makes the IO faster or not?

**Comment by JinZr** [comment]
Seems like this is an issue related to lhotse, you might want to raise an issue in lhotse's repo: https://github.com/lhotse-speech/lhotse
